tests/integration/targets/aci_l3out_extepg_to_route_control_profile/tasks/main.yml

```diff
@@ -0,0 +1,196 @@
+# Copyright: (c) 2022, Tim Cragg (@timcragg)
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI APIC host, ACI username and ACI password
+  fail:
+    msg: 'Please define the following variables: aci_hostname, aci_username and aci_password.'
+  when: aci_hostname is not defined or aci_username is not defined or aci_password is not defined
+
+- name: Set vars
+  set_fact:
+   aci_info: &aci_info
+    host: "{{ aci_hostname }}"
+    username: "{{ aci_username }}"
+    password: "{{ aci_password }}"
+    validate_certs: '{{ aci_validate_certs | default(false) }}'
+    use_ssl: '{{ aci_use_ssl | default(true) }}'
+    use_proxy: '{{ aci_use_proxy | default(true) }}'
+    output_level: debug
+
+# CLEAN ENVIRONMENT
+- name: Remove the ansible_tenant
+  aci_tenant: &tenant_absent
+    <<: *aci_info
+    tenant: ansible_tenant
+    state: absent
+
+# SETUP ENVIRONMENT
+- name: Create l3dom domain
+  cisco.aci.aci_domain: &domain_present
+    <<: *aci_info
+    domain: ansible_l3_dom
+    domain_type: l3dom
+    state: present
+
+- name: Add a new ansible_tenant
+  aci_tenant:
+    <<: *aci_info
+    tenant: ansible_tenant
+    description: Ansible tenant
+    state: present
+
+- name: Add a new l3out
+  aci_l3out:
+    <<: *aci_info
+    tenant: ansible_tenant
+    name: ansible_l3out
+    description: l3out for Ansible tenant
+    domain: ansible_dom
+    route_control: export
+    vrf: ansible_vrf
+    l3protocol: ospf
+    state: present
+
+- name: Add a new ExtEpg
+  aci_l3out_extepg:
+    <<: *aci_info
+    tenant: ansible_tenant
+    l3out: ansible_l3out
+    name: ansible_extEpg
+    description: ExtEpg for Ansible l3out
+    state: present
+
+- name: Add route control profile - ansible_rtctrl_profile
+  cisco.aci.aci_route_control_profile:
+    <<: *aci_info
+    tenant: ansible_tenant
+    l3out: ansible_l3out
+    profile: ansible_rtctrl_profile
+    prof_type: global
```

Comment:
prof_type is not a variable that is allowed as input, this should be profile_type as defined in your other created module

Reply:
fixed